website/blog/2022-08-01-aluminium.md

```diff
@@ -0,0 +1,355 @@
+---
+author: Rikito Taniguchi
+title: Metals v0.11.8 - Aluminium
+authorURL: https://twitter.com/<login>
+authorImageURL: https://github.com/<login>.png
+---
+
+We're happy to announce the release of Metals v0.11.8, which brings a number of improvements for both Scala 2 and Scala3.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">78</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">74</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">14</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">6</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/<num>?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- [Scala 3] Type completion
+- [Scala 3] Auto complete and import extension methods
+- [Scala 3] ConvertToNamedParameters code action
+- [Scala 3] ScaladocCompletion for Scala3
+- [Scala 3] Add interpolation completions
+- [Scala 2] Automatically import types in string interpolations
+- More stable code navigation across subprojects
+- Code Action documents
+- Support Scala 3.2.0-RC3, Scala 3.2.0-RC2
+
+and a lot of bugfixes!
+
+## [Scala 3] Type Competion
```

Comment:
@vzmerr Hey, Ina, could add some words about the Scala3 type completions with screen-recording?

Reply:
Yes, sure. Not sure about the screen recording though. Currently the completions keep crashing for me :( I would try and let you know.

Reply:
Is it on the newest version? That is very worrying, what errors are you getting?

Reply:
Oh, thank you for letting me know that, if it's a serious one, we may want to consider reverting the change for this version, and introduce polished type completion in the next release.

Reply:
Yes, it is on the newest version

Reply:
The error is `ERROR searching for `dotty/tools/dotc/util/SourcePosition#source.` failedjava.nio.channels.ClosedChannelException`

Reply:
That's the issue I fixed, please check if you are indeed in the newest version.

Reply:
Good to hear, it seems an already fixed issue. I appreciate if you could record a short demo video (because you know the most about this feature), but if you busy, let me know, I can take a video :)

Reply:
yes, sure, would be doing it right now.

Reply:
I was trying to produce this demo, but the completions are not working in consistency with the tests put on the PR. There are extra results getting produced also, on which the suffixed do not get added. I guess either there are still unresolved issues or this is just specific to my instance.

Reply:
Could you just show the examples that do work? It's your work, you should be able to showcase it.

Reply:
https://user-images.githubusercontent.com/109077949/182587035-5a044f54-b775-410e-9e8a-2f2888dc4241.mov

I wonder if this would be a good idea to only show the parts that are working

Reply:
It's fine, you can mention the cases where it is not working. The other alternative is not to mention it at all.

Reply:
I came up with this. Please give me feedback :) 
![Type Completion](https://user-images.githubusercontent.com/109077949/182594026-e80f6832-8084-42cd-9a33-b6aee92227e3.gif)

Reply:
Came up with this :) Please give me feedback
![Type Completion](https://user-images.githubusercontent.com/109077949/182594564-9c741030-d3d1-4d47-94c0-a5285ca71030.gif)